config.go

```diff
@@ -285,6 +286,7 @@ func parseCli(args []string) (options, []string, error) {
 	pflag.StringVarP(&purgeKeep, "purge-min-keep", "K", "0", "minimum number of dumps to keep when purging or 'all' to keep\neverything")
 	pflag.StringVar(&opts.PreHook, "pre-backup-hook", "", "command to run before taking dumps")
 	pflag.StringVar(&opts.PostHook, "post-backup-hook", "", "command to run after taking dumps\n")
+	pflag.BoolVar(&opts.UniformTimestamp, "uniform-timestamp", false, "Apply a single consistent timestamp to all filenames generated by pg_back instead of using individual file creation times")
```

Comment:
I don't find the word "consistent" great, as it is would imply that timestamps are incorrect without the option. It is not the case, since the default is to use a timestamp as close as possible to the time the data was dumped.

Also, please wrap the help message so that it is as long as the others.

Reply:
I have replaced `Apply a single consistent timestamp to all filenames generated by pg_back instead of using individual file creation times`

by `Use the same timestamp for all pg_back files instead of individual creation times`.